spec/bowlingGameSpec.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+describe('BowlingGame', function() {
+
```

Comment:
Overall, A good start to the testing. Working with baby steps is key to getting a full understanding of the challenge especially when working on your own